restapi.go

```diff
@@ -1060,7 +1060,7 @@ func (s *Session) GuildRoleEdit(guildID, roleID, name string, color int, hoist b
 		Name        string `json:"name"`        // The role's name (overwrites existing)
 		Color       int    `json:"color"`       // The color the role should have (as a decimal, not hex)
 		Hoist       bool   `json:"hoist"`       // Whether to display the role's users separately
-		Permissions int    `json:"permissions"` // The overall permissions number of the role (overwrites existing)
+		Permissions int64  `json:"permissions"` // The overall permissions number of the role (overwrites existing)
```

Comment:
To serialize as a string, the JSON tag should be `` `json:"permissions,string"` ``.